src/DssSpell.sol

```diff
@@ -20,211 +20,202 @@ import {Fileable, ChainlogLike} from "dss-exec-lib/DssExecLib.sol";
+        ClipperMomAbstract(CLIPPER_MOM).setPriceTolerance(MCD_CLIP_YFI_A, 50 * RAY / 100);
 
-        ClipAbstract(MCD_CLIP_LINK_A).upchost();
+        ClipAbstract(MCD_CLIP_YFI_A).upchost();
 
         // Replace flip to clip in the ilk registry
-        DssExecLib.setContract(ILK_REGISTRY, "LINK-A", "xlip", MCD_CLIP_LINK_A);
-        Fileable(ILK_REGISTRY).file("LINK-A", "class", 1);
-
-
-        // ------------------  CHAINLOG  -----------------
-
-        address log = DssExecLib.getChangelogAddress("CHANGELOG");
-
-        DssExecLib.setChangelogAddress("MCD_DOG", MCD_DOG);
-        DssExecLib.setChangelogAddress("MCD_END", MCD_END);
-        DssExecLib.setChangelogAddress("MCD_ESM", MCD_ESM);
-        DssExecLib.setChangelogAddress("CLIPPER_MOM", CLIPPER_MOM);
-        DssExecLib.setChangelogAddress("MCD_CLIP_LINK_A", MCD_CLIP_LINK_A);
-        DssExecLib.setChangelogAddress("MCD_CLIP_CALC_LINK_A", MCD_CLIP_CALC_LINK_A);
-        DssExecLib.setChangelogAddress("ILK_REGISTRY", ILK_REGISTRY);
-        ChainlogLike(log).removeAddress("MCD_FLIP_LINK_A");
-
-        DssExecLib.setChangelogVersion("1.3.0");
+        DssExecLib.setContract(ILK_REGISTRY, "YFI-A", "xlip", MCD_CLIP_YFI_A);
+        Fileable(ILK_REGISTRY).file("YFI-A", "class", 1);
+
+        DssExecLib.setChangelogAddress("MCD_CLIP_YFI_A", MCD_CLIP_YFI_A);
+        DssExecLib.setChangelogAddress("MCD_CLIP_CALC_YFI_A", MCD_CLIP_CALC_YFI_A);
+        ChainlogLike(CHANGELOG).removeAddress("MCD_FLIP_YFI_A");
+
+        // ------------- Stability fees -------------
+        DssExecLib.setIlkStabilityFee("LINK-A", FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("ETH-B", TEN_PCT, true);
+        DssExecLib.setIlkStabilityFee("ZRX-A", FOUR_PCT, true);
+        DssExecLib.setIlkStabilityFee("LRC-A", FOUR_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2DAIETH-A", THREE_PT_FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2USDCETH-A", FOUR_PT_FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("AAVE-A", THREE_PCT, true);
+        DssExecLib.setIlkStabilityFee("BAT-A", FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("MANA-A", THREE_PCT, true);
+        DssExecLib.setIlkStabilityFee("BAL-A", TWO_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2DAIUSDC-A", ONE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2LINKETH-A", FOUR_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2WBTCDAI-A", ZERO_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2AAVEETH-A", FOUR_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2DAIUSDT-A", THREE_PCT, true);
+
+        // ------------- Regular debt ceilings -------------
+
+        DssExecLib.decreaseIlkDebtCeiling("USDT-A", 25 * MILLION / 10, true);
+
+        // ------------- Auto line max ceiling changes -------------
+
+        DssExecLib.setIlkAutoLineDebtCeiling("YFI-A", 90 * MILLION);
+        // DssExecLib.setIlkAutoLineDebtCeiling("AAVE-A", 50 * MILLION);
```

Comment:
These commented out lines can be removed.

Reply:
I actually left them on purpose to be very explicit the ones changing the ceiling.